base_rest/components/service.py

```diff
@@ -102,6 +117,29 @@ def _secure_params(self, method, params):
         _logger.error("BadRequest %s", v.errors)
         raise UserError(_('Invalid Form'))
 
+    def _secure_response(self, method, response):
+        """
+        Internal method used to validate the output of the given method.
+        This response is validated according to a schema defined for the
+        method, with the convention '_validator_return_{method_name}'.
+        If the method is decorated with `@skip_secure_response` checks of
+        the response are skipped.
+        :param method: str or function
+        :param response: dict/json
+        :return: dict/json
+        """
+        method_name = method
+        if callable(method):
+            method_name = method.__name__
+        if hasattr(method, 'skip_secure_response'):
+            return response
+        schema = self._get_schema_output_for_method(method_name)
+        v = Validator(schema, purge_unknown=True)
```

Comment:
@acsonefho You can remove 'purge_unknown' for the response. We must be sure that the provided informations is conform to what is expected and in the same time avoid to lost time searching why a new field is not present into the response if the schema has not been adapted